utils/posttextparser.py

```diff
@@ -9,19 +10,19 @@
 
 
 # working good
-def posttextparser(obj,*,tried:bool=False) -> List[str]:
-    text:str = re.sub("\n", " ", obj)
-
+def posttextparser(obj, *, tried: bool = False) -> List[str]:
+    text: str = re.sub("\n", " ", obj)
     try:
         nlp = spacy.load("en_core_web_sm")
     except OSError as e:
         if not tried:
             os.system("python -m spacy download en_core_web_sm")
-            return posttextparser(obj,tried=True)
-        
-        print_step(" The spacy model can't load. You need to install it with the command \npython -m spacy download en_core_web_sm ")
+            time.sleep(5)
```

Comment:
why sleep ?

Reply:
Since the command needs some time to execute

Reply:
it is blocking code not async